lib/worker.js

```diff
@@ -19,11 +19,12 @@ const { createDirectories, cleanupDirectories } = require('./prepare');
 const { getSource, putRendition } = require('./storage');
 const Rendition = require('./rendition');
 
-const { imagePostProcess, needsImagePostProcess } = require('./postprocessing/image');
+const { imagePostProcess, needsImagePostProcess, prepareImagePostProcess } = require('./postprocessing/image');
```

Comment:
Extreme nitpick - to have them appear in the same order as they get used:
```suggestion
const { prepareImagePostProcess, needsImagePostProcess, imagePostProcess } = require('./postprocessing/image');
```